Python/bytecodes.c

```diff
@@ -4184,6 +4190,42 @@ dummy_func(
             frame->instr_ptr = (_Py_CODEUNIT *)instr_ptr;
         }
 
+        // Inlining prelude.
+        // Not too easy to express the stack effect.
+        op(_PRE_INLINE, (--)) {
+            // NULL out locals of the new inlined frame.
+            PyObject **end = frame->localsplus + oparg;
```

Comment:
This looks suspicious. IIUC the oparg here is the actual number of locals of the callee. But `frame` is the caller's frame (right?!) and so the correct `end` value is larger -- we'd need to add the size of the caller's locals *and* the offset on the stack of the first argument. I believe that the effect is that we don't zero out enough locals. It might be easier to adjust the argument in optimizer_bytecodes.c (`_PUSH_FRAME_INLINEABLE`).